File: Cpp/Strings/CountVowels.cpp
#include <iostream>
#include <string>
using namespace std;

int main()
{
    string str;
    int vowels = 0;

    cout << "Enter a string: ";
    getline(cin, str);

    for(char ch : str)
    {
        if(ch=='a'||ch=='e'||ch=='i'||ch=='o'||ch=='u'||
           ch=='A'||ch=='E'||ch=='I'||ch=='O'||ch=='U')
        {
            vowels++;
        }
    }

    cout << "Total Vowels = " << vowels;

    return 0;
}

File: Java/Strings/CountVowels.java
import java.util.Scanner;

public class CountVowels {

    public static int countVowels(String str) {

        int count = 0;

        str = str.toLowerCase();

        for (char ch : str.toCharArray()) {

            if (ch == 'a' || ch == 'e' || ch == 'i'
                    || ch == 'o' || ch == 'u') {

                count++;
            }
        }

        return count;
    }

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        System.out.print("Enter String: ");

        String input = sc.nextLine();

        System.out.println("Total Vowels = " + countVowels(input));

        sc.close();
    }
}